app/services/fact_importer.rb

```diff
@@ -164,7 +164,9 @@ def update_facts
     time = Time.now.utc
     updated = 0
     db_facts_names = []
-    db_facts.find_each do |record|
+    known_names = fact_names
```

Comment:
any reason to have this in another variable?

Reply:
Hmm not really, rebased.